src/main/java/uk/gov/ons/ctp/response/sample/endpoint/SampleEndpoint.java

```diff
@@ -275,6 +271,24 @@ public ResponseEntity<SampleUnitDTO> createSampleUnitsForSampleSummary(
     }
   }
 
+  @RequestMapping(
+      value = "/samplesummary/{sampleSummaryId}/check-all-units-present",
```

Comment:
While this checks _and_ transitions the sample summary status I can't think of anything that encapsulates both those distinct activities. Although as this should only ever be invoked _during_ a sample load, I wonder if it should be something really obvious like `check-(sample)-load-progress`?

Reply:
So change this to `"/samplesummary/{sampleSummaryId}/check-sample-load-progress"` ?
I can do that if that's what you're suggesting.